MICS6_Master.do

```diff
@@ -202,6 +205,10 @@ foreach name in $newMICS6countries {
 	mmerge hh1 hh2 ln using `wm'
 	drop _merge
 	
+	if inlist("`name'","Nepal2019") {
+	drop melevel2
```

Comment:
Could you clarify why dorp the variables here as we have the "keep" command at the very end that drop all the variables that are not needed?